SDK Showcase/SDK Showcase/AppState/UserState.swift

```diff
@@ -0,0 +1,10 @@
+//  Copyright © 2025 Onewelcome Mobile Identity. All rights reserved.
+
+extension AppState {
```

Comment:
why do you encapsulate UserState inside AppState?

Reply:
Ok, encapsulation removed.